packages/menu/src/MenuItem.ts

```diff
@@ -489,10 +589,18 @@ export class MenuItem extends LikeAnchor(
         this.active = false;
     }
 
-    public async openOverlay(): Promise<void> {
+    public async openOverlay(focus: boolean = false): Promise<void> {
```

Comment:
When navigating via keyboard, opening the menu should also set focus on an item within a menu.